samples/bluetooth/mesh/ble_peripheral_lbs_coex/src/lb_service_handler.c

```diff
@@ -119,7 +119,7 @@ static void lbs_adv_start(void)
 {
 	int err;
 	size_t id_count = 0xFF;
-
+	struct bt_le_adv_param adv_params = *BT_LE_ADV_CONN;
```

Comment:
Why do you create an instance of variable instead of pointer on it? Doesn't seem like optimization.

Reply:
https://github.com/nrfconnect/sdk-zephyr/commit/c51743c849618ce5e9b3ce841307ddcfdefad2ff#diff-bf17be4dee523f5115d7196112060ce2f51c5c1d475a62258eedca0f0f85b804R911

[BT_LE_ADV_PARAM ](https://github.com/nrfconnect/sdk-zephyr/blob/main/doc/releases/migration-guide-3.7.rst?plain=1#L804-L810)

Reply:
> Why do you create an instance of variable instead of pointer on it? Doesn't seem like optimization.

Saw your discussion about this in Teams. Now it seems reasonable.